src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -60,8 +60,7 @@
                         :color                    :colorId})
       (update :prodPreferredChainIds chain-ids-set->string)
       (update :testPreferredChainIds chain-ids-set->string)
-      (dissoc :watch-only?)
-      (dissoc :default-account?)))
+      (dissoc :watch-only? :default-account? :tokens :collectibles)))
```

Comment:
We have been unnecessarily sending huge amounts of tokens and collectible data whenever we edit an account.

Reply:
Nice!
I've faced some performance issues while just updating the profile!